src/Application/Actions/Donations/Create.php

```diff
@@ -238,21 +238,16 @@ protected function action(Request $request, Response $response, array $args): Re
 
     private function getStatementDescriptor(Charity $charity): string
     {
-        $prefix = 'Big Give';
-        $prefixIncSpace = "$prefix ";
-
-        $charityName = $charity->getName();
-
-        if ($charityName === null) {
-            return $prefix;
-        }
-
         $maximumLength = 22; // https://stripe.com/docs/payments/payment-intents#dynamic-statement-descriptor
+        $prefix = 'Big Give ';
 
-        return $prefixIncSpace . mb_substr(
-            $this->removeSpecialChars($charityName),
+        /**
+         * @psalm-suppress PossiblyNullArgument We don't expect to get here with no charity » let it crash for now.
```

Comment:
Yep, we're in strict types mode so if we don't have a charity name this [throws type error](https://3v4l.org/vCCDq). LGTM  to merge but IMHO we should keep the ticket where it is in Jira to do a more robust fix later.

Reply:
ah don't know why I was testing passing null to strlen anyway, we're passing null to `removeSpecialChars` which of course will also throw the error.